### ISSUE TITLE
Connected accounts guidance now invalid.

### ISSUE THREAD

**Issue by pcocodd** [MicrosoftDocs/OfficeDocs-o365seccomp#24]
I tried to test the guidance here: https://docs.microsoft.com/en-us/office365/securitycompliance/anti-spoofing-protection?redirectSourcePath=%252fen-us%252farticle%252fanti-spoofing-protection-in-office-365-d24bb387-c65d-486e-93e7-06a4f1a436c0#common-scenario-1---mailbox-forwarding

When I clicked Learn More in the Connected Accounts section of options (https://support.office.com/en-US/client/results?Shownav=true&lcid=1033&ns=OLWAO365B&version=15&omkt=en-US&ver=15&v=15.20.1059.23&mode=Mobile&helpid=ms.exch.owab.ConnectedAccounts), it shows that the Connected Accounts feature is being discontinued on September 15, 2018. Will the guidance be updated to suggest a different way that users can have their forwarded mail not trigger the phishing rules? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bab02396-69f6-5a83-fc27-7a2ba9ab78eb
* Version Independent ID: 63266ff4-ef2a-84b3-f1b9-c7b3e4d8a1b7
* Content: [Anti-spoofing protection in Office 365](https://docs.microsoft.com/en-us/office365/securitycompliance/anti-spoofing-protection?redirectSourcePath=%252fen-us%252farticle%252fanti-spoofing-protection-in-office-365-d24bb387-c65d-486e-93e7-06a4f1a436c0#common-scenario-1---mailbox-forwarding)
* Content Source: [SecurityCompliance/anti-spoofing-protection.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/anti-spoofing-protection.md)
* Service: **o365-administration**
* GitHub Login: @KCCross
* Microsoft Alias: **krowley**

**Comment by AndreaBarr** [comment]
@pcocodd Thank you for this feedback

I want to make sure this gets addressed and updated. I am sending this to the o365seccomp team. 

I was able to get this into the backlog for the O365seccomp team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
